compiler/AST/FnSymbol.cpp

```diff
@@ -735,10 +735,44 @@ TagGenericResult FnSymbol::tagIfGeneric(SymbolMap* map, bool abortOK) {
 // 'map' is expected to be non-NULL if this function has been instantiated.
 //
 bool FnSymbol::hasGenericFormals(SymbolMap* map) const {
+  bool anyGeneric = false;
+
+  std::vector<unsigned char> formalIsGeneric;
```

Comment:
I'd expect `formalIsGeneric` to contain bools.
OK to leave as-is.

Reply:
I was avoiding the `std::vector<bool>` which has a different representation and uses bit masking.